src/main/java/org/reactome/release/goupdate/GoTermInstanceModifier.java

```diff
@@ -194,12 +193,12 @@ public void updateGOInstance(Map<String, Map<String, Object>> goTerms, Map<Strin
 			}
 			catch (InvalidAttributeException | InvalidAttributeValueException e)
 			{
-				logger.error("Attribute/Value problem with \""+this.goInstance.toString()+ "\" " + e.getMessage());
+				logger.error("Attribute/Value problem with \""+this.goInstance.toString()+ "\" ", e);
```

Comment:
I feel like 'Attribute/Value problem' is a little generic here. Maybe actually having it say InvalidAttributeException would be better?

Reply:
It could be an `InvalidAttributeException` which could happen if you try to use an attribute that does not exist on a specific data-model type, or it could be `InvalidAttributeValueException` which could happen if the attribute itself is OK but you give it a bad value (such as string when only numeric is allowed). So just saying `InvalidAttributeException` would be misleading if the problem was actually with the value. Maybe I will break this up into two `catch`es with more specific messages...